frontend/src/components/App/App.js

```diff
@@ -1,60 +1,57 @@
-import React, { useState } from "react";
+import React, { useState, useCallback } from "react";
 import { GlobalStyle } from "./GlobalStyles";
-import MovieData from "../../mocks/Data";
 import Home from "../../pages/Home";
+import ErrorBoundary from "./ErrorBoundary";
+import { Provider } from "react-redux";
+import { BrowserRouter } from "react-router-dom";
 
-export default function App() {
-  const [movieData, setMovieData] = useState(MovieData);
+export default function App({ store }) {
   const [movieModal, setMovieModal] = useState(false);
```

Comment:
comment `You don't need to store all of that in App store. Use Redux for it. Remember the rule: if you have to pass props 2+ levels down, you need state management. https://blog.logrocket.com/the-upsides-of-prop-drilling-in-react/
'+ why do you need MovieData if you're fetching data from endpoint?` wasn't ONLY about movieData. it was about it and all of the above. Make your App stateless, you have Redux, you need state in not so many cases